### DISCUSSION TITLE
Unable to see assets following migration from Digital Ocean to AWS

### DISCUSSION THREAD

**Discussion by AndrewReid-Journey** [statamic/cms#7412] [Troubleshooting] [answered]
We previously stored all of our assets in a Digital Ocean bucket however we are in the process of migrating these over to S3 in AWS. I've set up the S3 bucket and copied over the assets and updated the AWS env variables for the site and it seems to work as the site loads as exepcted, but when i go to the Assets library in the Control Panel, I can only see asset items which I have uploaded after changing the location to the S3 bucket and nothing prior, even though those items exist in S3. This is what my asset library looks like in the control panel now : 
![image](https://user-images.githubusercontent.com/91478458/214977199-31778e48-656e-4bb1-a21c-5f1781853339.png)

but as you can see, I have a lot more on S3 which was previously uploaded : 

![image](https://user-images.githubusercontent.com/91478458/214977284-ab19baec-3bc9-4c05-8b28-207743048368.png)

Is there a way in which I can restore the existing tree structure into the media library following the migration?

Additional info : Our instance runs in a PHP container (php:8.1-fpm-alpine3.17) and we use a MySQL database to store our content. Statamic version is 3.3.61

**Comment by jasonvarga** [comment, marked as answer]
This has just been fixed in #7205 and will be in 3.3.66 which I just tagged. Should be available any second now.

**Reply by AndrewReid-Journey** [reply]
Thats brilliant, thanks so much for the rapid response. I'll try it out once its available.
